yarn.lock

```diff
@@ -14271,6 +14271,53 @@ trim-right@^1.0.1:
   resolved "https://registry.yarnpkg.com/trim-right/-/trim-right-1.0.1.tgz#cb2e1203067e0c8de1f614094b9fe45704ea6003"
   integrity sha1-yy4SAwZ+DI3h9hQJS5/kVwTqYAM=
 
+truffle-blockchain-utils@^0.0.10:
```

Comment:
What's up with these? Shouldn't they be the new names?

Reply:
Hehe, is it a dependency of a dependency?

Reply:
Yeah, I don't get it

Reply:
Well ethpm-registry has a dependency of truffle-contract.

Reply:
ah that would do it!